blocks-config/popup-builder/class-uagb-popup-builder.php

```diff
@@ -167,8 +167,9 @@ public function spectra_popup_shortcode( $attr ) {
 	 * @since 2.6.0
 	 */
 	public function enqueue_popup_scripts() {
-		$args   = array( 'post_type' => 'spectra-popup' );
-		$popups = new WP_Query( $args );
+		$query_args = array( 'post_type' => 'spectra-popup' );
+		$query_args = apply_filters( 'uagb_popup_query_args', $query_args );
```

Comment:
Since we are providing a filter it is recomended to check what kind of data is coming from filter. we should check for the data type.
```suggestion
		$query_args = array( 'post_type' => 'spectra-popup' );
		$query_args = apply_filters( 'uagb_popup_query_args', $query_args );
```

```suggestion
		$query_args = apply_filters( 'uagb_popup_query_args', array( 'post_type' => 'spectra-popup' ) );
```

Reply:
Thanks for suggesting. To make it simple and more readable format I update the code. Please review